packages/stream-tests/src/__tests__/multiquery.test.ts

```diff
@@ -0,0 +1,285 @@
+import tmp from 'tmp-promise'
+import type { Ceramic } from '../../../core/src/ceramic.js'
```

Comment:
```suggestion
import type { Ceramic } from '@ceramicnetwork/core'
```

And do similar thing on the following line. Cross-package imports are not allowed.